src/uu/basename/src/basename.rs

```diff
@@ -136,7 +136,7 @@ fn basename(fullname: &str, suffix: &str) -> String {
     }
 }
 
-#[allow(clippy::manual_strip)] // can be replaced with strip_suffix once the minimum rust version is 1.45
+// can be replaced with strip_suffix once the minimum rust version is 1.45
```

Comment:
Seems that it is still expected

```suggestion
#[allow(clippy::manual_strip)] // can be replaced with strip_suffix once the minimum rust version is 1.45
```